l10n/en-US/viewer.properties

```diff
@@ -280,3 +280,21 @@ editor_alt_text_save_button=Save
 editor_alt_text_decorative_tooltip=Marked as decorative
 # This is a placeholder for the alt text input area
 editor_alt_text_textarea.placeholder=For example, “A young man sits down at a table to eat a meal”
+
+# Editor resizers
+# LOCALIZATION NOTE (editor_resizer_label_topLeft): This is used in an aria label to help to understand the role of the resizer.
+editor_resizer_label_topLeft=Top left corner - resize
```

Comment:
I think you want to use — instead of -?  It's hard to tell the different in a monospace font.
https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-hyphens-and-dashes-c9

Reply:
Please remember to make the same changes in `web/l10n_utils.js` as well.